spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -598,6 +615,11 @@ def and_the_prompt_should_link_to_the_check_links_page
     end
   end
 
+  def but_when_I_edit_a_step
+    # Unable to do this through UI interaction without getting `ActiveRecord::RecordInvalid`
```

Comment:
Tried this:

```ruby
    click_on "Add a new step"
    fill_in "Step title", with: "Step title"
    fill_in "Content, tasks and links in this step", with: "Some content"
    click_on "Save step"
    expect(page).to have_content("Step was successfully created.")
    when_I_view_the_step_by_step_page
```

But I get:

```
     ActiveRecord::RecordInvalid:
       Validation failed: Title can't be blank, Base path can't be blank, Content can't be blank, Publishing app can't be blank, Schema name can't be blank
```

Then tried copying the stubbing (and its associated methods) from step_by_step_pages_controller_spec.rb:

```ruby
    allow(Services.publishing_api).to receive(:get_content)
      .with(step_by_step_page.content_id)
      .and_return(content_item(step_by_step_page))
```

But still get the same error message.